src/erlcloud_ec2.erl

```diff
@@ -1989,7 +1989,12 @@ extract_route(Node) ->
 extract_route_set(Node) ->
     [
      {destination_cidr_block, get_text("destinationCidrBlock", Node)},
-     {gateway_id, get_text("gatewayId", Node)},
+%%   {gateway_id, get_text("gatewayId", Node)},
```

Comment:
for the sake of simplicity I suggest we just have both
```
{gateway_id, get_text("gatewayId", Node)},
{nat_gateway_id, get_text("natGatewayId", Node)},
```

Reply:
done.